jupiterone/resource_rule.go

```diff
@@ -656,14 +655,17 @@ func (r *RuleModel) BuildCreateReferencedQuestionRuleInstanceInput() (client.Cre
 		return rule, err
 	}
 
-	// TODO: temp
-	labels, err := json.Marshal(r.Labels)
-	if err != nil {
-		return rule, err
-	}
-	err = json.Unmarshal(labels, &rule.Labels)
-	if err != nil {
-		return rule, err
+	if len(r.Labels) > 0 {
+		labels := make([]client.RuleInstanceLabelInput, len(r.Labels))
+
+		for i, label := range r.Labels {
+			labels[i] = client.RuleInstanceLabelInput{
+				LabelName:  label.LabelName.ValueString(),
+				LabelValue: label.LabelValue.ValueString(),
```

Comment:
Will this incorrectly convert things to a string if they are an int or a bool?

Reply:
We will only support strings in the terraform and do parsing if needed on the rule-service side. Terraform will actually throw an error if it receives anything but a string